server.go

```diff
@@ -63,19 +67,20 @@ func (s *RPCServer) handleWS(ctx context.Context, w http.ResponseWriter, r *http
 
 	c, err := upgrader.Upgrade(w, r, nil)
 	if err != nil {
-		log.Error(err)
-		w.WriteHeader(500)
+		log.Errorw("upgrading connection", "error", err)
+		// note that upgrader.Upgrade will set http error if there is an error
```

Comment:
Unrelated fix?

Reply:
Fixes golang logs mentioned in the issue - https://github.com/filecoin-project/lotus/issues/9988#issuecomment-1380750483